core/templates/domain/feedback_message/ThreadMessage.model.ts

```diff
@@ -26,8 +26,11 @@ export interface ThreadMessageBackendDict {
   'entity_id': string;
   'message_id': number;
   'text': string;
-  'updated_status': string;
-  'updated_subject': string;
+  // Status and Subject for frontend instances of thread message
+  // domain objects should be null initially.These properties should
```

Comment:
Do not include "initially". As we have discussed elsewhere, having a temporary value on initialization is not a sufficient reason to allow null, in general.

Instead, explain constructively what a null value for either property means conceptually. That is sufficient. (In this comment, the idea is not to justify the use of null, it's more to provide guidance to developers about when to use it and what it means in this context.)

Reply:
updated